spec/javascripts/fixtures/NewsletterSticky.html

```diff
@@ -1,9 +1,11 @@
 <div id="fixture-1">
   <div class="news-signup-sticky news-signup-sticky--positioned" data-dough-component="NewsletterSticky">
     <form onsubmit="return false;">
-      <button class="news-signup-sticky__close">
+      <button class="news-signup-sticky__close" id="first-focusable">
```

Comment:
There's a pattern within the ruby applications (certainly within RAD and codebases which use site prism) which have javascript tests to us classes with a 't-' prefix, eg.

https://github.com/moneyadviceservice/rad/blob/master/spec/javascripts/tests/MultiTableFilter_spec.js#L9
https://github.com/moneyadviceservice/rad/blob/master/spec/javascripts/fixtures/MultiTableFilter.html#L19

I think the intention is so that in the event of ids / classes being refactored/removed, it's more obvious that the particular element is under test by something. I don't know if this is a pattern within the frontend codebase, but it's maybe something that could be a discussion point at the next frontend meeting?